cmd/init.go

```diff
@@ -771,10 +779,26 @@ func validateFlags() []string {
 		}
 	}
 
+	userGateway := false
+	for i, gateway := range requiredUserGateway {
+		if v.IsSet(gateway) && v.GetString(gateway) != "" {
+			if userGateway {
+				errors = append(errors, fmt.Sprintf("provide only one of %v", requiredUserGateway))
```

Comment:
Nit: change error message to "Found value for both %v, expected only one or the other to be set."

Reply:
The intent of my suggestion is to just make it more clear that CSI found definitions for both.
Like I said, it's a nit.